main.go

```diff
@@ -146,6 +147,7 @@ func main() {
 		make([]webhook.Webhook, 0),
 		ingress.Webhook(utils.InCapsuleGroup(capsuleGroup, ingress.Handler())),
 		pvc.Webhook(utils.InCapsuleGroup(capsuleGroup, pvc.Handler())),
```

Comment:
Do we need the `InCapsuleGroup` Middleware?

I guess a Pod would be started by a ReplicaSet or a DaemonSet and this would be ignored.